include/tins/ip_address.h

```diff
@@ -204,8 +205,11 @@ class TINS_API IPv4Address {
 namespace std {
 
 template<>
-TINS_API struct hash<Tins::IPv4Address> {
-    size_t operator()(const Tins::IPv4Address& addr) const;
+struct hash<Tins::IPv4Address> {
+    size_t operator()(const Tins::IPv4Address& addr) const
+    {
+        return std::hash<string>()(addr.to_string());
```

Comment:
This used to perform the hash of the `uint32_t` value, can you roll it back so hashing an address is allocation free? (note to self, the same should be done for `IPv6Address`). Also, minor syntax suggestion, can you move the brace here so it looks like:

```) const { // < that one```